reference-page/components/global-navigation.md

```diff
@@ -147,7 +147,7 @@
 			<div class="wds-global-navigation__notifications-menu wds-dropdown">
 				<div class="wds-dropdown__toggle wds-global-navigation__dropdown-toggle">
 					<div class="wds-global-navigation__notifications-menu-counter">3</div>
-					<svg class="wds-icon wds-icon-small" alt="notifications">
```

Comment:
I'd keep this class here
if we'll go back to 24x24 as a default we would have to re-add it here and in all apps that implement it